pkg/controller/operators/catalog/step_ensurer.go

```diff
@@ -300,3 +300,26 @@ func (o *StepEnsurer) EnsureUnstructuredObject(client dynamic.ResourceInterface,
 	status = v1alpha1.StepStatusPresent
 	return
 }
+
+// EnsureConfigMap writes the specified ConfigMap object to the cluster.
+func (o *StepEnsurer) EnsureConfigMap(namespace string, configmap *corev1.ConfigMap) (status v1alpha1.StepStatus, err error) {
```

Comment:
hmm, couldn't we use the dynamic client for all resources going forward?

Reply:
We should probably, yes. Right now we have a `Ensure()` method for every type that OLM supports in the `StepEnsurer` but we can probably pack everything into the dynamic client logic instead. Would result in one very large function versus several smaller ones though